pangeo-notebook/environment.yml

```diff
@@ -20,6 +20,7 @@ dependencies:
  - dask-ml
  - datashader
  - descartes
+ - duckdb
```

Comment:
From reading https://github.com/conda-forge/duckdb-feedstock/issues/6 and https://github.com/conda-forge/python-duckdb-feedstock/pull/105, it seems like `duckdb` is simply an alias for `python-duckdb`, so maybe use `python-duckdb` to be clear that this is only the Python bindings.

Did you want to pull in `duckdb-cli` as well from https://github.com/conda-forge/duckdb-split-feedstock?

Reply:
Thanks I added both.